client/cli/src/commands/run_cmd.rs

```diff
@@ -404,7 +404,7 @@ pub fn is_node_name_valid(_name: &str) -> std::result::Result<(), &str> {
 		return Err("Node name should not contain invalid chars such as '.' and '@'")
 	}
 
-	let invalid_patterns = r"^https?:\/\/";
+	let invalid_patterns = r"^https?:/";
```

Comment:
```suggestion
	let invalid_patterns = r"^https?:";
```

Anything starting with `http:` or `https:` is already bad, or?  
Btw we do not check whether the name is non-empty 🙈